src/Components/DisplayToDoList.js

```diff
@@ -18,11 +18,11 @@ const DisplayToDoList = (props) => {
                                 <input
```

Comment:
Like Esther said, remember to add a form for your inputs

Reply:
Now added!